includes/templates/class-user-resend-email-verify-page.php

```diff
@@ -0,0 +1,61 @@
+<?php
+/**
+ * User resend email verify page template.
+ *
+ * @package HivePress\Templates
+ */
+
+namespace HivePress\Templates;
+
+use HivePress\Helpers as hp;
+
+// Exit if accessed directly.
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * Resend email verification page.
+ */
+class User_Resend_Email_Verify_Page extends Page_Narrow {
```

Comment:
If we already have the Email Verify page, maybe there's a way to combine these two into one (e.g. changing the message and hiding the form based on condition)? Let me know in tg.